scripts/flash/designated-report.js

```diff
@@ -13,34 +14,41 @@ var getPrivateKeyFromString = require("../dp/lib/get-private-key").getPrivateKey
  * Move time to Market end time and do initial report
  */
 function designateReportInternal(augur, marketId, outcome, userAuth, invalid, auth, callback) {
-  augur.markets.getMarketsInfo({ marketIds: [marketId] }, function (err, marketsInfo) {
-    var market = marketsInfo[0];
-    var marketPayload = { tx: { to: marketId } };
-    // make sure we have the correct account to report with
-    if (userAuth.address !== market.designatedReporter && auth.address === market.designatedReporter) {
-      userAuth = auth;
+  var amount = 10000;
```

Comment:
@bthaile Should this value be hardcoded?

Reply:
hardcoding is fine for this application, nice to have a high default value.